views/site/partials/_books.php

```diff
@@ -7,6 +7,10 @@
 
 <div class="books-list">
 <?php foreach ($books as $i => $book): ?>
+    <?php if ($i !== 0 && $i % 2 === 0): ?>
+        <div class="clearfix visible-sm"></div>
```

Comment:
Can't it be moved bottom as the following?

```php
<?php if ($i !== 0 && $i % 2 === 0): ?>
        <div class="clearfix visible-md visible-sm"></div>
    <?php endif ?>
```

Reply:
@samdark update, thanks